src/main/java/org/myrobotlab/service/Solr.java

```diff
@@ -463,6 +467,34 @@ public IplImage bytesToImage(byte[] bytes) throws IOException {
     // Scala
   }
 
+  /**
+   * deserialize from a png byte array to a base64 encoded string
+   * for display inline in html.
+   * 
+   * @param bytes
+   *          input bytes
+   * @return a string
+   * @throws IOException
+   *           boom
+   * 
+   */
+  public String bytesToBase64Jpg(byte[] bytes) {
+    //
+    // let's assume we're a buffered image .. those are serializable :)
+    ByteArrayOutputStream os = new ByteArrayOutputStream();
+    try {
+      BufferedImage bufImage = ImageIO.read(new ByteArrayInputStream(bytes));
+      ImageIO.write(bufImage, "jpg", os);
+      os.close();
+    } catch (IOException e) {
+      // TODO Auto-generated catch block
+      return "ERROR converting image to jpg base64.";
+    }
+
+    String data = String.format("data:image/%s;base64,%s", "jpg", Base64.getEncoder().encodeToString(os.toByteArray()));

```

Comment:
:+1:  would be nice in CodeUtils

Reply:
we have org.myrobotlab.image.Util  .. this is an image specific util.. moved it there.